python/pyspark/ml/tuning.py

```diff
@@ -266,7 +269,7 @@ class CrossValidatorModel(Model, ValidatorParams):
     """
     .. note:: Experimental
 
-    Model from k-fold cross validation.
+    CrossValidatorModel model returns the best set of estimators for your model.
```

Comment:
Java docs will be generated from the scala doc, so no need for anything special here.